go.mod

```diff
@@ -34,6 +34,8 @@ require (
 	k8s.io/kubernetes v1.23.0
 )
 
+require github.com/stretchr/testify v1.7.0
```

Comment:
assuming this was separated out for a reason @coreydaley ?

if so, can you put a comment here as to why?

Reply:
Weird, go mod must have done that. I have merged it into the larger require section and pushed the commit.
Good catch!